perses/rjmc/geometry.py

```diff
@@ -1513,17 +1691,21 @@ def __init__(self, reference_system, growth_indices, global_parameter_name='grow
         # Create new System
         from simtk import openmm
         growth_system = openmm.System()
+        atoms_with_positions_system = openmm.System()
```

Comment:
It would be useful to document what the difference between `growth_system` and `atoms_with_positions_system` is here in comments!